lib/shared/store.js

```diff
@@ -305,40 +312,44 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
     }
 
     case ACTIONS.SELECT_DRIVE: {
-      if (!action.data) {
+      const device = action.data
```

Comment:
Sorry to keep banging on about consistency (but hey at least I'm consistent :wink: ), but on line 166 you have `const drives` but here you have `const device` :confused:

Reply:
`action.data` is used in a polymorphic manner and here it's a device field `String`, whereas on L166 it's a list of drives: `Array<Object>`. We could potentially change the field used in `action` and is definitely worth looking into in another issue/PR.

Reply:
Right. Can you see why I'm getting confused between 'drives' and 'devices' though? :slightly_smiling_face: 
(perhaps more verbose comments is all that's needed to clear things up?)

Reply:
I wish we had TS here and that'd clarify it in the type :P